vnext/Microsoft.ReactNative/ReactInstanceSettings.idl

```diff
@@ -215,6 +215,10 @@ namespace Microsoft.ReactNative
     DOC_DEFAULT("8081")
     UInt16 SourceBundlePort { get; set; };
 
+    DOC_STRING("Flag controlling whether the bundle URL requests inline source maps.")
```

Comment:
For the other options that only affect when using the bundler server, we start the description with `When using a @.UseFastRefresh, @.UseLiveReload, or @.UseWebDebugger`, which makes it clear that it wouldn't affect the generated bundle file.

Reply:
I also like the more info in the snapshot property description about when you might want to turn this off. -- And this property is arguably the more important one to describe, since this is where the user will set the option.